x-pack/plugins/ml/public/components/anomalies_table/influencers_cell.js

```diff
@@ -88,10 +78,22 @@ export class InfluencersCell extends Component {
   }
 
   render() {
+    const recordInfluencers = this.props.influencers || [];
+
+    const influencers = [];
+    _.each(recordInfluencers, (influencer) => {
```

Comment:
nit, for consistency with other array loops in this PR, this could be `forEach`.
It'd also make it obvious that `recordInfluencers` is an array and `influencer` is an object